java/bulk-import/bulk-import-runner/src/test/java/sleeper/bulkimport/runner/BulkImportJobDriverIT.java

```diff
@@ -170,6 +174,14 @@ void shouldImportDataSinglePartition(BulkImportJobRunner runner) throws IOExcept
                     record = reader.read();
                 }
                 assertThat(recordsInThisFile).isSortedAccordingTo(new RecordComparator(getSchema()));
+                assertThat(SketchesDeciles.fromFile(schema, fileReference, sketchesStore))
+                        .isEqualTo(SketchesDeciles.builder()
+                                .field("key", deciles -> deciles
+                                        .min(0).max(99)
+                                        .rank(0.1, 10).rank(0.2, 20).rank(0.3, 30)
+                                        .rank(0.4, 40).rank(0.5, 50).rank(0.6, 60)
+                                        .rank(0.7, 70).rank(0.8, 80).rank(0.9, 90))
+                                .build());
```

Comment:
This seems to imply there's just one file in the bulk import result, otherwise this assertion would fail? Please can we make this more explicit? It doesn't look like there's a need for a for loop here. 

It doesn't look like there's a need to link this to reading the records from the file, inside the try statement like this. It might be worth extracting a helper record to read the records.

One option would be to use `assertThat(stateStore.getFileReferences()).singleElement().satisfies(fileReference -> {...});`.